messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
+^^^^^^^^^^^^^
 
-Only valid if all the following hold:
+For a ``LockedTransfer`` to be considered valid the following conditions have to be true.
 
 - There is a channel which matches the given :term:`chain id`, :term:`token network` address, and :term:`channel identifier`.
 - The corresponding channel is in the open state.
 - The :term:`nonce` is increased by ``1`` in respect to the previous :term:`balance proof`
 - The :term:`locksroot` must change, the new value must be equal to the root of a new tree, which has all the previous locks plus the lock provided in the message.
-- The :term:`locked amount` must increase, the new value is equal to the old value plus the lock's amount.
-- The lock's amount must be smaller then the participant's :term:`capacity`.
+- Have the new lock represented in merkleroot.
+- The :term:`locked amount` must increase by exactly ``lock.amount`` otherwise, the message would be rejected by the recipient. If the ``locked_amount`` is increased by more, then funds may get locked in the channel. If the ``locked_amount`` is increased by less, then the recipient will reject the message as it may mean it received the funds with an on-chain unlock. The initiator will stipulate the fees based on the available routes and incorporate it in the lock's amount. Note that with permissive routing it is not possible to predetermine the exact `fee` amount, as the initiator does not know which nodes are available, thus an estimated value is used..
+- The lock's amount must be smaller than the current :term:`capacity`. If the amount is higher then the recipient will reject it, as it means he will be spending money it does not own. .
 - The lock expiration must be greater than the current block number.
 - The :term:`transferred amount` must not change.
 
-Fields
-^^^^^^
+Matrix message
```

Comment:
Maybe we can put the message specification before the preconditions? We refer in the preconditions section to the various data fields but did not introduce them. Or is there a reason why you put that in this order?

Reply:
Is it clear what we mean by Matrix message? Maybe we can define that as the message and clarify in the first sentence that this message is sent over our Matrix communication Layer

Reply:
The below should not be matrix-specific so i agree with leaving out "matrix"